the-open-university-harvard.csl

```diff
@@ -158,10 +158,7 @@
   <macro name="edition">
     <choose>
       <if is-numeric="edition">
-        <group delimiter=" ">
-          <number variable="edition" form="ordinal"/>
-          <text term="edition" form="short"/>
-        </group>
+        <number variable="edition" form="ordinal" suffix=" edn"/>
```

Comment:
why are you doing this? Always preferable to use terms (which you can define above if they need to differ from default) so that, e.g. people can more quickly change this style to a different locale

Reply:
I didn't realize I could define terms for the default locale. Is my new commit the right way to do that? Thanks!